sagemaker-experiments/tensorflow2-california-housing-regression-experiment/tensorflow2-california-housing-regression-experiment.ipynb

```diff
@@ -0,0 +1,445 @@
+{
```

Comment:
<p>Is it possible to use a prefix in the SageMaker default bucket so that a customer can run the example even if they can't or don't want to create a new bucket?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2589/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Fixed to use the default SageMaker bucket.